app/build.gradle

```diff
@@ -33,4 +33,5 @@ dependencies {
     compile 'com.squareup.picasso:picasso:2.5.2'
     compile 'com.squareup.retrofit2:retrofit:2.1.0'
     compile 'com.squareup.retrofit2:converter-gson:2.1.0'
+    compile 'com.android.support.constraint:constraint-layout:1.0.2'
```

Comment:
This is back again? 😅  You should review your own code before asking for reviews so this kind of things don't slip